data/campaigns/The_South_Guard/scenarios/08b_The_Tides_of_War.cfg

```diff
@@ -384,13 +390,13 @@
             [then]
                 [message]
                     speaker=Minister Hylas
-                    message= _ "I have summoned the great Council of Westin!"
+                    message= _ "The council of Westin has summoned some aid from our allies to the north!"
```

Comment:
Capitalize Council

And, again, north? The Elves? They're allies. The King? He's a bit more than an ally. Check the maps.